_docs/2.0.0/usage/menas-conformance-rules.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+title: Usage - Menas Conformance Rules
+version: '2.0.0'
+categories:
+    - '2.0.0'
+    - usage
+---
+
+## Table Of Contents
+<!-- toc -->
+- [Table Of Contents](#table-of-contents)
+- [Intro](#intro)
+- [Casting Conformance Rule](#casting-conformance-rule)
+- [Concatenation Conformance Rule](#concatenation-conformance-rule)
+- [Drop Conformance Rule](#drop-conformance-rule)
+- [Literal Conformance Rule](#literal-conformance-rule)
+- [Mapping Conformance Rule](#mapping-conformance-rule)
+- [Negation Conformance Rule](#negation-conformance-rule)
+- [SingleColumn Conformance Rule](#singlecolumn-conformance-rule)
+- [SparkSessionConf Conformance Rule](#sparksessionconf-conformance-rule)
+- [Uppercase Conformance Rule](#uppercase-conformance-rule)
+- [FillNulls Conformance Rule](#fillnulls-conformance-rule)
+<!-- tocstop -->
+
+## Intro
+
+Conformance Rules are a way for the user to enhance the data. Every conformance rule has an output column and an option for running a control measure. Output column defines the name of the output column into which the rule will output. Control measure check will run an Atum control measure check as in previouse stages as defined in controls of `_INFO` file.
+
+Every column also has one or more input columns.
+
+**Important** - We never override a column. Each rule produces a new column.
+
+## Casting Conformance Rule
+
+Allows users to cast a specific column to another type. This conformance rules provides a selection of other types.
+
+## Concatenation Conformance Rule
+
+Concatenation rule concatenates multiple input columns together into a single column. Columns are frist transformed into their string representation.
+
+## Drop Conformance Rule
+
+Drop Conformance rule remove a column from the DataFrame.
```

Comment:
Maybe just Remove a column from the DataFrame.

Reply:
You mean omit the name?

Reply:
Yes, seems like repeating

Reply:
It is consistent with every other entry. I find it easier to read a full sentence

Reply:
Would it sound better if we say 'Dataset" instead of "dataframe"

Reply:
Somewhere else I. removed it to output data.